### ISSUE TITLE
Same query parameters for the pricehistory endpoint, with different results

### ISSUE THREAD

**Issue by goshlanguage** [rideron89/tda-sdk-rs#2]
Hey there,

Thank you for your work on this repo. 

I'm noticing that when I set query params in the TD Ameritrade API tool, I'm getting different results than when I setup the same parameters with a `GetPriceHistoryParams` object and was wondering if you might have any insight?

For example, if I go to the [TD Ameritrade API tool](https://developer.tdameritrade.com/price-history/apis/get/marketdata/%7Bsymbol%7D/pricehistory) and enter the following parameters, I get a full year's candle data, confirmed by cross referencing the oldest epoch with an epoch time converter for ease:

```
            end_date: None
            frequency_type: daily
            frequency: 1
            need_extended_hours_data: None
            period_type: year
            period: 1
            start_date: None
```

However, when I use the same set of parameters with the `GetPriceHistoryParams` struct, it seems as if the defaults are used for `start_date` and `end_date` in that, I see in the comments that the default is for one trading day:

```
            end_date: None,
            frequency_type: Some(String::from("daily")),
            frequency: Some(String::from("1")),
            need_extended_hours_data: None,
            period_type: Some(String::from("year")),
            period: Some(String::from("1")),
            start_date: None,
``` 

I am a bit green to rust, so I'm sorry if I'm missing something fairly obvious, but I'm curious why this happens? 

I don't see any defaults implemented in the [`get_price_history` method](https://github.com/rideron89/tda-sdk-rs/blob/master/src/lib.rs#L252-L301) and I don't see anything in the [params.rs](https://github.com/rideron89/tda-sdk-rs/blob/master/src/params.rs) that would set a default, unless the `Default` impl is used if the fields are none and that's a language mechanic that I'm overlooking?

**Comment by rideron89** [comment]
Hello and thanks for creating this ticket! It looks like I goofed and had the query params for the `get_price_history` call wrong. I was using underscores when I should have been using camelCase, causing the TDA API to use their defaults.

I published a new version (v0.1.1) that fixes this, so if you upgrade then it should work as expected.

**Comment by goshlanguage** [comment]
Thanks @rideron89
